src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingMethodSymbol.cs

```diff
@@ -148,16 +148,18 @@ public override ImmutableArray<CustomModifier> ReturnTypeCustomModifiers
             get
             {
                 return RetargetingTranslator.RetargetModifiers(
-                    _underlyingMethod.ReturnTypeCustomModifiers,
-                    ref _lazyCustomModifiers);
+                    _underlyingMethod.ReturnTypeCustomModifiers, _underlyingMethod.RefCustomModifiers,
+                    ref _lazyCustomModifiers).TypeCustomModifiers;
             }
         }
 
-        internal override ushort CountOfCustomModifiersPrecedingByRef
+        public override ImmutableArray<CustomModifier> RefCustomModifiers
         {
             get
             {
-                return _underlyingMethod.CountOfCustomModifiersPrecedingByRef;
+                return RetargetingTranslator.RetargetModifiers(
+                    _underlyingMethod.ReturnTypeCustomModifiers, _underlyingMethod.RefCustomModifiers,
+                    ref _lazyCustomModifiers).RefCustomModifiers;
```

Comment:
Please extract a private `CustomModifiersTuple` property, for use here and above, to avoid having two distinct places where `_lazyCustomModifiers` is set. Same comment for `RetargetingParameterSymbol` and `RetargetingPropertySymbol`.